bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractOptionsFieldImpl.java

```diff
@@ -51,6 +52,18 @@ public boolean isEnforceEnum() {
         return enforceEnum;
     }
 
+    public String[] removeDuplicates(boolean flag, int length) {
+        String[] enumArray = this.enums;
```

Comment:
enums could also be other data types like number, boolean etc. Why are you using string here